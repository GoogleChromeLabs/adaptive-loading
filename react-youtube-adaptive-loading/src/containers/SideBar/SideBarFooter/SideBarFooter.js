import React from 'react';
import './SideBarFooter.scss'

export function SideBarFooter() {
  return (
    <React.Fragment>
      <div className='footer-block'>
        <div>About Press Copyright</div>
        <div>Creators Advertise</div>
        <div>Developers +MyTube</div>
        <div>Legal</div>
      </div>
      <div className='footer-block'>
        <div>Terms Privacy</div>
        <div>Policy & Safety</div>
        <div>Test new features</div>
      </div>
      <div className='footer-block'>
        <div><a href='https://github.com/marvtron/youtube-react' target='_blank' rel='noreferrer noopener'>View GitHub repo</a></div>
      </div>
      <div className='footer-block'>
        <div>Â© marvtrondev - A Youtube clone for educational purposes under fair use.</div>
      </div>
    </React.Fragment>
  );
}