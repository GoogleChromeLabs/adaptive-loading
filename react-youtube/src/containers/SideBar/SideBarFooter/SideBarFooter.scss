.footer-block {
    padding-bottom: 10px;
    padding-left: 16px;
    color: #918888;
  }