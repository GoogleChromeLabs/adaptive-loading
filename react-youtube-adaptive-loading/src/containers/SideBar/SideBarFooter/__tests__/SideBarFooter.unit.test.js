import React from 'react';
import {SideBarFooter} from '../SideBarFooter';
import {shallow} from 'enzyme';

describe('SideBarFooter', () => {
  test('renders', () => {
    const wrapper = shallow(<SideBarFooter/>);
    expect(wrapper).toMatchSnapshot();
  });
});