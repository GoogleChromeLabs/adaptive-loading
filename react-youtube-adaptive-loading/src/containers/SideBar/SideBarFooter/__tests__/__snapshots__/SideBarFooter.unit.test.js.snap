// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarFooter renders 1`] = `
<Fragment>
  <div
    className="footer-block"
  >
    <div>
      About Press Copyright
    </div>
    <div>
      Creators Advertise
    </div>
    <div>
      Developers +MyTube
    </div>
    <div>
      Legal
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Terms Privacy
    </div>
    <div>
      Policy & Safety
    </div>
    <div>
      Test New Features
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      <a
        href="https://github.com/marvtron/youtube-react"
        rel="noopener noreferrer"
        target="_blank"
      >
        GitHub Repo
      </a>
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Â© marvtrondev - A Youtube clone for educational purposes under fair use.
    </div>
  </div>
</Fragment>
`;
